set.mm

```diff
@@ -75703,9 +75703,9 @@ be a set (so instead it is a proper class).  Here we prove the denial of
       UTVOVPWGVNVKDVBVAVCVDVEVLVMVNVFVJVKBVGVHVI $.
   $}
 
-  $( The successor of an ordinal number is an ordinal number.  Proposition 7.24
-     of [TakeutiZaring] p. 41.  (Contributed by NM, 6-Jun-1994.)  (Proof
-     shortened by BTernaryTau, 30-Nov-2024.) $)
+  $( The successor of an ordinal number is an ordinal number.  Closed form of
+     ~ onsuci .  Proposition 7.24 of [TakeutiZaring] p. 41.  (Contributed by
+     NM, 6-Jun-1994.)  (Proof shortened by BTernaryTau, 30-Nov-2024.) $)
   suceloni $p |- ( A e. On -> suc A e. On ) $=
```

Comment:
Maybe we can relabel this theorem ~sucelon, and change the current label ~sucelon to ~sucelonbi.

Reply:
I chose a slightly different labeling, but along the same idea.